components/config/src/config/markup.rs

```diff
@@ -27,6 +41,8 @@ pub struct ThemeCss {
 pub struct Markdown {
     /// Whether to highlight all code blocks found in markdown files. Defaults to false
     pub highlight_code: bool,
+    /// Emit either "error" or "warn" for missing highlight languages. Defaults to "warn".
+    pub highlight_missing_level: HighlightLanguageLevel,
```

Comment:
I'm wondering if we can just do `error_on_unknown_highlight`. Do we have anything other than error or warn?

Reply:
I was going for symmetry with `internal_level` in the link checker. I don't have a strong preference.

I suppose we could also have a "do nothing" level, if users are annoyed by the warnings.

Reply:
I wouldn't hide the warnings in any way. Let's go with a bool for now

Reply:
Done!